demo/Views/StyleManagerView.vala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2024 elementary, Inc. (https://elementary.io)
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+ */
+
+public class StyleManagerView : Gtk.Box {
+    construct {
+        var label = new Gtk.Label (_("Override the application theme:"));
+
+        var dont_button = new Gtk.CheckButton.with_label (_("Use system theme")) {
+            active = true
+        };
+
+        var force_light = new Gtk.CheckButton.with_label (_("Force light theme")) {
+            group = dont_button
+        };
+
+        var force_dark = new Gtk.CheckButton.with_label (_("Force dark theme")) {
```

Comment:
We actually don't translate the demo since it's only used by developers. Also probably better to be consistent with language here and use "style" which is used in Onboarding and Settings. We don't usually use the word "theme" because historically it means something quite different that we don't support

```suggestion
        var label = new Granite.HeaderLabel ("Visual Style");

        var dont_button = new Gtk.CheckButton.with_label ("Follow system setting") {
            active = true
        };

        var force_light = new Gtk.CheckButton.with_label ("Light") {
            group = dont_button
        };

        var force_dark = new Gtk.CheckButton.with_label ("Dark") {
```